app/assets/javascripts/controllers/vm_cloud/vm_cloud_remove_security_group_form_controller.js

```diff
@@ -0,0 +1,40 @@
+ManageIQ.angular.app.controller('vmCloudRemoveSecurityGroupFormController', ['$http', 'vmCloudRemoveSecurityGroupFormId', 'miqService', function($http, vmCloudRemoveSecurityGroupFormId, miqService) {
+  var vm = this;
+
+  var init = function() {
+    vm.afterGet = false;
+    vm.vmCloudModel = {
+      security_group: null,
+    };
+    vm.security_groups = [];
+    vm.formId = vmCloudRemoveSecurityGroupFormId;
+    vm.model = "vmCloudModel";
+    vm.saveable = miqService.saveable;
+    miqService.sparkleOn();
+    $http.get('/vm_cloud/remove_security_group_form_fields/' + vmCloudRemoveSecurityGroupFormId).then(function(response) {
```

Comment:
Oh, also.. isn't this something that should be supported by the API?

I wonder if it even makes sense to add this functionality the old way - if it's already in the API, no need for this, and if it's not, why wasn't the API change done first?